app/assets/javascripts/listings/ListingController.js.coffee

```diff
@@ -39,6 +39,7 @@ ListingController = (
   $scope.lotterySearchNumber = ''
   $scope.smallDisplayClass = "small-display-none"
   $scope.lotteryRankingSubmitted = false
+  $scope.loadingLotteryResults = false
```

Comment:
I'm wondering if we should do the ng-disabled stuff for forms/buttons to be just a general `processingSubmit` type thing as opposed to creating unique one for each type of form. thoughts?